util/image_viewer/slip_viewer/tile_generation.py

```diff
@@ -58,7 +59,6 @@ def get_flex_image_multipanel(
 
     from iotbx.detectors import generic_flex_image
     from libtbx.test_utils import approx_equal
-    from xfel.cftbx.detector.metrology import get_projection_matrix
```

Comment:
Ah, I see this moved from DIALS to dxtbx in https://github.com/cctbx/dxtbx/pull/422/. As per comment there, it would be preferable for dxtbx to not depend on xfel, so we can work towards a separately distributable dxtbx